components/core/src/util.rs

```diff
@@ -8,6 +8,15 @@ pub mod win_perm;
 
 use std::mem;
 
+#[cfg(windows)]
+pub fn pwsh_args(command: &str) -> Vec<&str> {
```

Comment:
Nice consolidation... it would be nice to have some documentation comments about why this is important, though.

Reply:
good call. comment added!